horovod/ray/elastic_v2.py

```diff
@@ -162,6 +162,11 @@ class ElasticParams(BaseParams):
         reset_limit (int): Maximum number of times that the training
             job can scale up or down the number of workers after
             which the job is terminated.
+        cooldown_range(Tuple[int, int]): Range of seconds(min, max) a failing
```

Comment:
What do you want to express with the notation `seconds(min, max)` here? Or is there just a whitespace missing before the `(`?

The same further down.

Reply:
I wanted to express that the first element if the range implies the min and the second element implies the max.  Thinking over it, that's the definition of a range, so I'll remove that notation.